packages/preview/genealotree/0.1.0/typst.toml

```
@@ -0,0 +1,12 @@
+[package]
+name = "genealotree"
+version = "0.1.0"
+entrypoint = "genealotree.typ"
+authors = ["DrLoiseau"]
+license = "GPL-3.0-only"
+description = "A package to draw genealogical trees, based on CeTZ"
+repository = "https://codeberg.org/drloiseau/genealogy"
+keywords = ["draw", "cetz", "genealogical", "genealogy", "tree"]
+categories = ["visualization"]
+disciplines = ["anthropology", "biology", "history", "medicine"]
+compiler = "0.10.0"
```

The example directory is generally excluded, as it is not that useful to have it in the local package cache.
```suggestion
compiler = "0.10.0"
exclude = [ "examples" ]
```